util/getopt.h

```diff
@@ -33,6 +33,11 @@
 #ifdef __clang__
 #warning Working around bug in LLVM optimizer
 #warning For more details see https://bugs.llvm.org/show_bug.cgi?id=27190
+#define GETOPT_USE_GOTO
+#endif
+
+#ifdef GETOPT_USE_GOTO
+/* Workaround with goto. */
```

Comment:
I'd say `computed goto` here -- longjmp is arguably a type of goto, but the key issue is that *computed* gotos aren't standard-compliant.